apps/frontend/src/app/pages/company/gpts/gpts.component.html

```diff
@@ -0,0 +1,347 @@
+<div class="container mx-auto min-h-screen px-3 sm:px-0">
+  <div class="pb-4 pt-6 flex flex-wrap justify-between gap-6">
+    <h2 class="font-sans font-medium text-4xl text-base-100">GPTs</h2>
```

Comment:
![image](https://github.com/Venture-Miner/wagademy/assets/111710515/8c7668cc-416f-468c-aa97-f509af7fb4ee)

I can't make the drag and drop work. When I try uploading something to it it just doesn't react at all